test/sign_mpe_funs.js

```diff
@@ -52,6 +52,27 @@ function isValidSignatureClaim(contractAddress, channelId, nonce, amount, signat
         ethereumjsutil.stripHexPrefix(expectedSigner).toLowerCase();
 }
 
+function getVFromSignature(signature) {
+  signature = signature.substr(2); //remove 0x
+  const v = '0x' + signature.slice(128, 130);    // Should be either 27 or 28
+  const v_decimal = web3.toDecimal(v) ;
+  if(v_decimal < 27 ) 
+    return v_decimal + 27;
+  else 
+    return v_decimal;
+}
+
+function getRFromSignature(signature) {
+  signature = signature.substr(2); //remove 0x
+  const r = '0x' + signature.slice(0, 64);
+  return r;
+}
+
+function getSFromSignature(signature) {
+  signature = signature.substr(2); //remove 0x
+  const s = '0x' + signature.slice(64, 128);
+  return s;
+}
```

Comment:
I would propose to have one function: splitSignatureToVRS

Reply:
Thought we can reuse these functions individually... but i dont any one calling these explicitly. Added one function to return all the three values.